elements/drag-drop-elements/pl-drag-drop-element.css

```diff
@@ -0,0 +1,86 @@
+.side-by-side {
+  --width: 355px;
+  --margin-bottom: 0px;
+}
+
+.top-and-bottom {
+  --width: 370px;
+  --margin-bottom: 50px;
+}
```

Comment:
I think hardcoded widths here is still not a good idea--what is PrairieLearn changed the size of the question panel at some point?

It seems like doing it by percentages is a more robust solution. For top-and-bottom, the width could be 100%, for side-by-side, it could be calc(50% - 5px) or something like that. Does that work?

Reply:
Fixed, I had to initially hardcode the value because I forgot the dropzones were nested inside a div, and so percentage styling to the dropzones were percentages of the parent div, not the question panel

now fixed properly